lib/checks/text-alternatives/link-text.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "link-text",
```

Comment:
This could use a more descriptive name. It's checking for unique link text, right? I would think something like `unique-link-text` would be more clear. WDYT @WilcoFiers ?